index.js

```diff
@@ -91,47 +91,65 @@ OpenDKIMVerifyStream.prototype._build_result = function(error) {
   return result;
 };
 
-OpenDKIMVerifyStream.prototype.write = function(buf) {
-  if (buf && buf.length && !(this.finished)) {
-    var chunk = this._get_chunk(buf);
-    try {
-      this.opendkim.chunk({
+OpenDKIMVerifyStream.prototype.write = function (buf) {
+  var self = this;
```

Comment:
I think technically you don't need this, since `=>` functions don't overwrite `this`.  I'm not opposed to it, tho', since it's easy to mix `function()`s and `=>`s still, and we have no linting to catch it.  Just wanted to mention it in case you didn't notice.